src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -124,20 +161,21 @@ def job_id_list(self):
             return self.rq_data[JOB_ID_LIST]
         raise JobIDException(JOBS_NOT_PROVIDED_ERR)
 
+    @property
+    def batch_id(self):
+        if BATCH_ID in self.rq_data:
+            return self.rq_data[BATCH_ID]
+        raise JobIDException(BATCH_NOT_PROVIDED_ERR)
+
+    def has_batch_id(self):
+        return BATCH_ID in self.rq_data
+
     @property
     def cell_id_list(self):
         if CELL_ID_LIST in self.rq_data:
             return self.rq_data[CELL_ID_LIST]
         raise ValueError(CELLS_NOT_PROVIDED_ERR)
 
-    def input(self):
-        if JOB_ID in self.rq_data:
-            return (JOB_ID, self.job_id)
-        elif JOB_ID_LIST in self.rq_data:
-            return (JOB_ID_LIST, self.job_id_list)
-        else:
-            return None
```

Comment:
this is now unneeded -- it was used when generating errors to show what the input to a function was. Now the whole request is included in the error, so this is surplus to requirements.